packages/ecc-client-ga4gh-tes/package.json

```diff
@@ -1,63 +1,57 @@
 {
   "name": "@elixir-cloud/tes",
-  "description": "Web components for TES",
+  "description": "Web Component for interactive with TES API",
+  "license": "MIT",
+  "version": "0.0.0",
+  "type": "module",
   "main": "./dist/index.js",
   "module": "./dist/index.mjs",
   "types": "./dist/index.d.ts",
-  "type": "module",
-  "version": "0.0.2",
-  "license": "MIT",
-  "repository": "https://github.com/elixir-cloud-aai/cloud-components",
+  "componentsPrefix": "ecc-client-lit-ga4gh-tes-",
   "exports": {
-    ".": "./dist/index.js",
-    "./package.json": "./package.json"
+    ".": { "types": "./dist/index.d.ts", "import": "./dist/index.js" },
+    "./dist/custom-elements.json": "./dist/custom-elements.json",
+    "./dist/index.js": "./dist/index.js",
+    "./dist/components/*": "./dist/components/*",
+    "./dist/react": "./dist/react/index.js",
+    "./dist/react/*": "./dist/react/*"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/elixir-cloud-aai/cloud-components.git"
   },
-  "files": [
-    "images",
-    "dist"
-  ],
-  "keywords": [
-    "TES",
-    "Task Execution Service",
-    "Elixir",
-    "Cloud Component",
-    "Web Component"
-  ],
   "scripts": {
-    "dev": "npm run build && concurrently -k -r \"npm run build -- --watch\" \"wds\"",
-    "lint:fix": "npm run lint -- --fix",
-    "lint": "eslint --ext .ts,.html . --ignore-path .gitignore",
+    "analyze": "cem analyze --litelement",
+    "build": "node ../../scripts/build.js",
```

Comment:
how about adding clean command just before this, IK build script is handeling this but clearly not efficiently.
PS: accordingly lets update the template files as well (handlebars).

Reply:
Hey @SalihuDickson, if possible can you maybe make PR from your branch to this branch? It will ease the merging & testing all together!